apps/src/code-studio/components/Libraries/LibraryCreationDialog.jsx

```diff
@@ -1,12 +1,34 @@
 /*global dashboard*/
 import React from 'react';
 import PropTypes from 'prop-types';
-import BaseDialog from '../../../templates/BaseDialog';
+import Dialog, {Body} from '@cdo/apps/templates/Dialog';
 import {connect} from 'react-redux';
 import {hideLibraryCreationDialog} from '../shareDialogRedux';
 import libraryParser from './libraryParser';
 import LibraryClientApi from './LibraryClientApi';
 import i18n from '@cdo/locale';
+import PadAndCenter from '../../../templates/teacherDashboard/PadAndCenter';
+import {Heading1, Heading2} from '../../../lib/ui/Headings';
```

Comment:
Might be good to use @cdo format for these two as well given the three levels of ../.

Reply:
:heavy_check_mark: